src/index.ts

```diff
@@ -22,64 +23,67 @@ const defaultMinifyOptions: HtmlMinifierTerserOptions = {
   minifyCSS: true,
 };
 
-export default function simpleHtmlPlugin({ minify: minifyOptions = false, inject }: Options = {}) {
+export default function simpleHtmlPlugin({ minify: minifyOptions = false, inject, order = "pre" }: Options = {}) {
   return {
     name: 'vite:simple-html',
-    async transformIndexHtml(html: string) {
-      const { data, tags } = inject || {};
+    transformIndexHtml: {
+      order,
+      handler: async (html: string) => {
+        const { data, tags } = inject || {};
 
-      let result = html;
+        let result = html;
 
-      // Replace ejs variables
-      if (data) {
-        Object.entries(data).forEach(([key, value]) => {
-          result = result.replaceAll(new RegExp(`<%= ${key} %>`, 'g'), value);
-        });
-      }
+        // Replace ejs variables
+        if (data) {
+          Object.entries(data).forEach(([key, value]) => {
+            result = result.replaceAll(new RegExp(`<%[=-] ${key} %>`, 'g'), value);
```

Comment:
I wonder if that's really the intended behavior. `<%=` is for replacing single variables, while `<%-` is more for... Embedding stuff?

Reply:
This is not that important to me. I can change my writing style to adapt to this plug-in. What really hinders me from using it is the following problem. If it cannot be handled before vite parse HTML, the plug-in cannot execute normally in my scenario.